yarn.lock

```diff
@@ -0,0 +1,12739 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Usually, we do not commit lock files. This will cause for conflict issues when pulling/pushing the changes.

Reply:
Added into `.gitignore`